portal/integrated-windows-authentication/README.md

```diff
@@ -1,47 +1,41 @@
-# Integrated windows authentication
+# Integrated Windows Authentication
 
-Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+Connect to an IWA secured portal and search for maps.
 
-![](IntegratedWindowsAuthentication.png)
+![Image of integrated windows authentication](IntegratedWindowsAuthentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), allows users to easily authenticate for different services using their domain's Windows credentials. This simplifies logging in to services that are provided through an intranet environment, as only a single username and password combination are required for the applications that support IWA.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
```

Comment:
```suggestion
Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications, but isn't always practical for internet apps.
```